templates/index.html

```diff
@@ -209,6 +210,19 @@ <h1>
     </div>
   </paper-dialog>
 
+  <paper-dialog id="join-epschedule" class="join-dialog">
+    <div id="GithubUpdateBox">
+      <h2>JOIN EPSCHEDULE</h2>
+      <p>Click "About" for more details.</p>
+      <p>We meet every Thursday at 11:10 in TMAC-007.</p>
+      <p>If you have any suggestions for potential features, please click the three dots in the left panel and click "Report a bug".</p>
+      <a href = "https://forms.microsoft.com/r/sEqxAVt4PB">Join Us</a>
+      <p> Check us out: </p>
+      <button onclick="window.location.href = 'https://github.com/EastsidePreparatorySchool/epschedule'" class="githubbutton"><img src="../static/images/github.svg"></button>
```

Comment:
should this just be an &lt;a&gt; element rather than a button? (same for the button below)